### PR TITLE
Ignored null values from ExampleBuilder

### PR DESCRIPTION
**Pull request by richard-walsh-sb**
Add unit test to demonstrate behavior where null values returned from `ExampleBuilder` are either ignored and returned as the default `"string"` value or as string of value `"null"`

### CODE REVIEW

File: src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java [hunk 1]
```diff
@@ -944,4 +945,57 @@ public void testSwos126() throws Exception {
...
+        ParseOptions options = new ParseOptions();
+        options.setResolve(true);
+        options.setResolveFully(true);
+
+        OpenAPI openAPI = new OpenAPIV3Parser().read("src/test/swagger/null-examples-oas3.yaml", null, options);
+
+        ApiResponse response = openAPI.getPaths().get("/object-with-null-example").getGet().getResponses().get("200");
+        Example example = ExampleBuilder.fromSchema(response.getContent().get("application/json").getSchema(), null, ExampleBuilder.RequestType.READ);
+        String output = Json.pretty(example);
+        assertNull(output);
```
Comment (gracekarina) [inline src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java:961]: Hi, the expected result for the example here should be `foo: null` instead of the example being null

Comment (hkosova) [reply src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java:961]: @gracekarina no. The object itself has `null` as the example value:
```yaml
    ObjectWithNullExample:
      type: object
      properties:
        foo:
          type: string
      nullable: true
      example: null   # <----
```

Comment (gracekarina) [reply src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java:961]: the ExampleBuilder will build an example, of the `foo` property and it may be null,
that's why `foo: null`;

Comment (gracekarina) [reply src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java:961]: @hkosova the parser will return this: 
```
    ObjectWithNullExample:
      type: object
      properties:
        foo:
          type: string
      nullable: true
      example: null   # <----
```
But  inflector's Example Builder will try to recreate an example of the property according to the type.
